aws/sqs_event_manager/sqs_event_manager/app.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2025 SUSE LLC.  All rights reserved.
+#
+# This file is part of suse-saas-tools
+#
+# suse-saas-tools is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# mash is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with mash.  If not, see <http://www.gnu.org/licenses/>
+#
+"""
+Lambda to handle SQS messages from from
+AWS Metering/Entitlement Marketplace API's
+"""
+import json
+import logging
+
+from sqs_event_manager.queue import delete_message
+from sqs_event_manager.message import AWSSNSMessage
+
+logger = logging.getLogger()
```

Comment:
I figured for the logging to CloudWatch you have to set the log level to INFO. It's a bit strange and took me some time to find out. So you need to add

```python
logger = logging.getLogger('sqs_event_manager')
logger.setLevel('INFO')
```

really the text "INFO" not the logging.INFO variables. It was the only way to get cloud watch to include the logging done by the lambda